jerry-main/CMakeLists.txt

```diff
@@ -59,7 +59,11 @@ endmacro()
 # Jerry with libfuzzer support
 if(JERRY_LIBFUZZER)
   jerry_create_executable("jerry-libfuzzer" "libfuzzer.c")
-  target_link_libraries("jerry-libfuzzer" jerry-port-default -fsanitize=fuzzer)
+  if(JERRY_OSS_FUZZ)
+    target_link_libraries("jerry-libfuzzer" jerry-port-default)
+  else()
+    target_link_libraries("jerry-libfuzzer" jerry-port-default -fsanitize=fuzzer)
+  endif()
```

Comment:
I'd suggest reducing duplication. If I'm not mistaken `target_link_libraries` can be called multiple times and that will not overwrite previous items but concatenate them. (From the cmake docs: "Repeated calls for the same `<target>` append items in the order called.") Thus, something like the following might work (already using the on/oss/off approach):

```cmake
target_link_libraries("jerry-libfuzzer" jerry-port-default)
if(NOT (JERRY_LIBFUZZER STREQUAL "OSS"))
  target_link_libraries("jerry-libfuzzer" -fsanitize=fuzzer)
endif()
```